rosbag2_transport/test/rosbag2_transport/test_record_all_use_sim_time.cpp

```diff
@@ -50,10 +51,10 @@ TEST_F(RecordIntegrationTestFixture, record_all_with_sim_time)
 
   rosbag2_transport::RecordOptions record_options =
   {
-    true, true, {string_topic, clock_topic}, "rmw_format", 100ms
+    false, false, {string_topic, clock_topic}, "rmw_format", 100ms
```

Comment:
Why was it needed to change the options here?

Reply:
The first `false` option telling recorder to not record all available topics rather to recorder only those which we specified in list {string_topic, clock_topic}.
The second one `false` telling recorder to run discovery tread if topics hasn't been visible during `Recorder` construction phase.  It could be in some cases and this why I put extra steps further on to make sure that `Recorder` ready to record those topics before we will start publishing. We need discovery to be on in general case. Discovery thread will stop automatically as soon as Recorder will subscribe to the all topics from options list.

Reply:
Thanks for the explanation.